README.md

```diff
@@ -89,3 +89,32 @@ https://github.com/users/zuev93/projects/1
 
 # Contacts
 If you have any suggestions or questions feel free to create discussion or issue in GitHub or join our discord server https://discord.gg/3TQ97w8Qwq
+
+# How to develop
+
+1. Install `Visual Studio 2022`
+2. Clone the Oni Multiplayer repository
+3. If you do not have a custom `steam library` location for `Oxygen Not Included` skip to `step 4`
+   - Copy/paste the file `Directory.Build.props` inside the `root` folder, and rename the copy to `Directory.Build.props.user`
+   - Erase all contents, and fill it in with the following (example for `D` drive)
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+    <PropertyGroup>
+        <SteamLibraryPath>D:\SteamLibrary</SteamLibraryPath>
+    </PropertyGroup>
+</Project>
+```
+
+4.  Open the `OniMod.sln` file inside of the `root` folder
+5. If you do not have the multiplayer mod subscribed skip to `step 6`, else you will have 2 entries ingame, so lets make that identifiable: to the right side of solution explorer, go to `MultiplayerMod -> mod.yaml`
```

Comment:
The if else is a little confusing to read in english. I would break this up into two sentences.

_5.) If you do not have the multiplayer mod subscribe skip to step 6. If you do have it installed you will have 2 entries in game, so lets distinguish them:_

Reply:
Ajusted with your recommendation